suite-native/atoms/src/Button/Button.tsx

```diff
@@ -278,11 +307,15 @@ export const Button = ({
                 ]}
             >
                 <HStack alignItems="center">
-                    {iconLeft && icon}
+                    <ButtonAccessoryView element={viewLeft} iconColor={iconColor} iconSize={size} />
```

Comment:
Why was the conditional rendering removed? IMO it is more readable like that.

Reply:
0c52c10957